inputs/radiation_advection.pin

```diff
@@ -105,6 +105,7 @@ absorption = true
 do_nu_electron = true
 do_nu_electron_anti = false
 do_nu_heavy = false
+src_solver = oned
```

Comment:
I'd like to hang onto the 4D solver for now in case there are parameter regimes where it is more accurate. One could imagine using it as a first attempt when `Pr >> Pg` or something. For now `oned` should be the hardcoded default, though.

Reply:
:+1: